examples/salesforce/index.js

```diff
@@ -0,0 +1,108 @@
+const express = require('express');
+const jsforce = require('jsforce');
+const everyauth = require('../..');
+
+const app = express();
+const port = 3000;
+const serviceId = 'sfdc';
+const userId = 'usr-123'; // normally determined through authorization
+const tenantId = 'contoso'; // organization/project userId is part of
+
+app.get('/', (req, res) => {
+  res.redirect('/integrations');
+});
+
+// The management page that shows connected services
+app.get(`/integrations`, async (req, res) => {
+  // Check if the user's organization has already authorized access to SFDC
+  const credentials = await everyauth.getIdentity(serviceId, { userId, tenantId }); // TODO: this should only be tenantId
```

Comment:
I think we're missing an opportunity here to teach the user that the tenantId / userId can be used from the query string after handling the **finishedUrl**, check **github-oauth** example for ideas.

Reply:
Relying on the values passed as query params to the finishedUrl creates an identity spoofing opportunity - I was actually going to separately suggest we remove those query params. Imagine Adam and Eve, both valid users of our customer's app, where Eve tricks Adam into visiting {finishedUrl}?userId=eve. Soon enough Adam is sending his stock trade confirmations to Eve's Slack.